model/src/main/java/uk/gov/pay/commons/api/json/PayZonedDateTimeSerializer.java

```diff
@@ -0,0 +1,29 @@
+package uk.gov.pay.commons.api.json;
+
+import com.fasterxml.jackson.core.JsonGenerator;
+import com.fasterxml.jackson.databind.PropertyNamingStrategy;
+import com.fasterxml.jackson.databind.SerializerProvider;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+import com.fasterxml.jackson.databind.ser.std.StdSerializer;
+
+import java.io.IOException;
+import java.time.ZonedDateTime;
+
+import static uk.gov.pay.commons.model.ApiResponseDateTimeFormatter.ISO_INSTANT_MILLISECOND_PRECISION;
+
+@JsonNaming(PropertyNamingStrategy.SnakeCaseStrategy.class)
+public class PayZonedDateTimeSerializer extends StdSerializer<ZonedDateTime> {
```

Comment:
Maybe this class could be named `ApiResponseDateTimeSerializer` to match `ApiResponseDateTimeFormatter`?

It’s conceivable we could end up using different date/time formats for things that aren’t API responses so we probably don’t want a really generic name.